Assets/Scripts/Utils/PathFinder.cs.meta

```diff
@@ -0,0 +1,11 @@
+fileFormatVersion: 2
```

Comment:
Maybe we should add *.meta to .gitignore . Unity generates this automatically, right?

Reply:
I've been reading about this topic, and some people say it's better to ignore `.meta`s only if we ignore the source of them... We could try tough, if any problem surface we'll know it's because of that